packages/platform-android/native_modules.gradle

```diff
@@ -245,12 +211,12 @@ class ReactNativeModules {
 
 def autoModules = new ReactNativeModules(logger)
 
-ext.applyNativeModulesSettingsGradle = { DefaultSettings defaultSettings, String root = ".." ->
-  autoModules.applySettingsGradle(defaultSettings, root)
+ext.applyNativeModulesSettingsGradle = { DefaultSettings defaultSettings ->
+  autoModules.addReactNativeModuleProjects(defaultSettings)
 }
 
-ext.applyNativeModulesAppBuildGradle = { Project project, String root = ".." ->
```

Comment:
let's keep the root (unused) in gradle methods for backwards compatibility, output a warning that it's deprecated and remove it in v4.